src/app/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -1,9 +1,10 @@
 import React, { useState, useEffect, useContext } from 'react';
+import { oneOfType, func, shape, any } from 'prop-types';
 import Banner from './Banner/index.canonical';
 import consentBannerUtilities from './CanonicalLogic';
 import { UserContext } from '#contexts/UserContext';
 
-const Canonical = () => {
+const Canonical = ({ onDismissFocusRef, ...props }) => {
```

Comment:
Is there a reason for spreading prop here? It's not really clear what they are

Reply:
Thanks @HarveyPeachey,
I had initially used `...props` because at one point there was widespread use of `forwardRef` and props are a necessary parameter for this, but I can remove this now 👍 Nice spot